03_Modules/Ocpi/src/modules/temp/exceptions/global.exception.handler.ts

```diff
@@ -0,0 +1,27 @@
+import { FastifyError, FastifyReply, FastifyRequest } from 'fastify';
+import { ILogObj, Logger } from 'tslog';
+import {
+  ExceptionHandler,
+  HttpStatus,
+  UnauthorizedException,
+} from '@citrineos/base';
+import { NotFoundException } from './not.found.exception';
+
+export class GlobalExceptionHandler implements ExceptionHandler {
```

Comment:
Setting a `GlobalExceptionHandler` via `setErrorHandler` in `EverythingElseApi` for now, but should be repurposed to have a more robust error handling setup